src/input/__snapshots__/input.spec.js.snap

```diff
@@ -379,6 +450,9 @@ exports[`common/Input graphical (JSX) types should print an input (checkbox) 1`]
   >
     {"name":"checkbox","props":{"input":{"name":"name","value":"","checked":false},"meta":{"active":false,"asyncValidating":false,"autofilled":false,"dirty":false,"form":"testForm","invalid":false,"pristine":true,"submitting":false,"submitFailed":false,"touched":false,"valid":true,"visited":false},"className":"","placeholder":"","disabled":false,"type":"checkbox","children":null}}
   </div>
+  <div
```

Comment:
The error message div is always printed ? 
This is intended ?

Reply:
yes fot opacity effect, like a label

Reply:
Ok I understand that you play with opacity